src/main/kotlin/hr/foi/pop/backend/controllers/UserController.kt

```diff
@@ -38,10 +44,46 @@ class UserController {
 
     }
 
+    @PatchMapping("/{userId}/activate")
+    fun activateUser(
+        @PathVariable userId: String,
+        @RequestBody request: ActivateUserRequestBody
+    ): ResponseEntity<*> {
+        return try {
+            request.activated!!
+
+            val activateOrDeactivate: ActivateUserDefinitions = ActivateUserDefinitions.getEnumByCode(request.activated)
```

Comment:
Thanks for info.

Comment no.1 - resolved. 

Comment no.2 - Hmmm did not consider this. I will then extract userId checker to controller. I thought userService should handle this kind of errors. Then I suppose we should add new error code for this particular case, because it is not bad body (so this message does not look good to me).

Can I also edit the issue (do I have your permission), or should I leave it all to you?

Reply:
Even though it is passed via URI path param?

Reply:
Ohhhhhh. Right... I completely disregarded that part, sorry. Sure, then it is not a bad body. In that case, shouldn't it simply be 404?

Reply:
I always tend to think of REST services as going through folders and files. If the path is "/api/v2/users/1421/activate", but the folder 1421 doesn't exist, it's as if the entire path/route doesn't exist. Just as if user has requested a "/api/v2/lolol/lolol/3213/lol". Right?